python/tk_multi_demo/syntax_highlighter.py

```diff
@@ -82,8 +82,8 @@ def __init__(self, document, palette):
         # Multi-line strings (expression, flag, style)
         # FIXME: The triple-quotes in these two lines will mess up the
         # syntax highlighting from this point onward
-        self.tri_single = (QtCore.QRegExp("'''"), 1, self._style("string2"))
-        self.tri_double = (QtCore.QRegExp('"""'), 2, self._style("string2"))
+        self.tri_single = (QtCore.QRegularExpression("'''"), 1, self._style("string2"))
+        self.tri_double = (QtCore.QRegularExpression('"""'), 2, self._style("string2"))
```

Comment:
I believe I already covered this on #36

Reply:
If this is the only change in this PR, you can close it. Changing from `QRegExp` to `QRegularExpression` requires additional changes on the multi-line parser.